src/lib/remark-plugins/rewrite-tutorial-links/index.ts

```diff
@@ -39,8 +54,7 @@ const learnProductOptions = Object.keys(ProductOption).join('|')
 const learnLink = new RegExp(
   `(learn.hashicorp.com)|(/(collections|tutorials)/(${learnProductOptions}|cloud)/)|^/(${learnProductOptions}|cloud)$`
 )
-
-let TUTORIAL_MAP
+const docsLink = new RegExp(`(${Object.values(PRODUCT_DOCS_PATHS).join('|')})`)
```

Comment:
Thinking about making this tighter, but there's a greater variety of sub-paths with docs so I'm not sure exactly how to capture all of that.

Reply:
This seems like a reasonable "first-pass" filter 👍